### PR TITLE
classic Actor 用のAPIを非推奨に変更

### PR DESCRIPTION
**Pull request by tksugimoto**
## 関連
- https://github.com/lerna-stack/lerna-app-library/issues/11

### CODE REVIEW

**Review by negokaz — APPROVED** [review]
LGTM 👍

File: CHANGELOG.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   - Improve documentation
   - Provide [migration guide](doc/migration-guide.md)
 - Update to `ScalaTest 3.1.4` from `ScalaTest 3.0.9`
+- Deprecate API for classic Actor. The API for classic Actor will be removed in the next major version upgrade.
```
Comment (negokaz) [inline CHANGELOG.md:31]: @tksugimoto `ScalaTestWithClassicActorTestKit` は非推奨にしなくても良いでしょうか？

[lerna-app-library/ScalaTestWithClassicActorTestKit.scala at untyped-api-is-deprecated · lerna-stack/lerna-app-library](https://github.com/lerna-stack/lerna-app-library/blob/untyped-api-is-deprecated/lerna-testkit/src/main/scala/lerna/testkit/akka/ScalaTestWithClassicActorTestKit.scala)

Comment (tksugimoto) [reply CHANGELOG.md:31]: b6ec2d39c0b49738bb01aab4b7a151c9b8e90c37 `chore: ScalaTestWithClassicActorTestKit を非推奨に変更`

**Review by negokaz — APPROVED** [review]
LGTM 👍

**Pull request merged**